src/DataGrid.tsx

```diff
@@ -219,7 +221,9 @@ export interface DataGridProps<R, SR = unknown, K extends Key = Key> extends Sha
    * Toggles and modes
    */
   /** @default true */
-  enableVirtualization?: Maybe<boolean>;
+  enableVirtualization?: Maybe<boolean | VirtualizationOptions>;
```

Comment:
```suggestion
  enableVirtualization?: Maybe<'row' | 'column' | boolean>;
```
I think we can keep it simple for now.